lib/crawlexir/search/keyword.ex

```diff
@@ -3,19 +3,24 @@ defmodule Crawlexir.Search.Keyword do
   import Ecto.Changeset
 
   alias Crawlexir.Auth.User
+  alias Crawlexir.Search.Report
 
   schema "keywords" do
     field :keyword, :string
 
     belongs_to :user, User
+    has_one :report, Report
 
     timestamps()
   end
 
+  @required_field ~w(keyword)a
```

Comment:
I saw in the report, we put this on the top. Maybe we move this one too 👍

Reply:
Right, it's better indeed. Fixed in 4071a198995f83efbc52b1586f838bf48f342c50